src/data/markdown/docs/01 guides/02 Using k6/14 Scenarios.md

```diff
@@ -0,0 +1,712 @@
+but please [create a bug issue](https://github.com/loadimpact/k6/issues/new?labels=bug&template=bug_report.md)
+if that's not the case.
+
+There are several benefits of using scenarios in your tests:
+- Multiple scenarios can be declared in the same script, and each one can
+  execute a different JavaScript function, which makes organizing tests easier
+  and more flexible.
+- Every scenario can use a distinct VU and iteration scheduling pattern,
+  powered by a purpose-built [executor](#executors). This enables modeling
+  of advanced execution patterns which can better simulate real-world traffic.
+- They can be configured to run in sequence or parallel.
+- Different environment variables and tags can be set per scenario.
+
+If you're excited to try it out, make sure you're using
+[k6 v0.27.0 or above](https://github.com/loadimpact/k6/releases) and keep reading!
+Note that scenarios are supported both in k6 CLI (local) and
+[k6 Cloud](/cloud) execution.
+
+
+## Configuration
+
+Scenarios are primarily configured via the `options` object in your test script. For example:
+
+<div class="code-group" data-props='{"labels": [], "lineNumbers": "[false]"}'>
+
+```js
+export let options = {
+  scenarios: {
+    my_cool_scenario: {  // arbitrary and unique scenario name, will appear in the result summary, tags, etc.
+      executor: 'shared-iterations',  // name of the executor to use
+      // common scenario configuration
+      startTime: '10s',
+      gracefulStop: '5s',
+      env: { EXAMPLEVAR: 'testing' },
+      tags: { example_tag: 'testing' },
+      // executor-specific configuration
+      vus: 10,
+      iterations: 200,
+      maxDuration: '10s',
+    },
+    another_scenario: { ... }
+  }
+}
+```
+
+</div>
+
+
+### Common configuration
+
+| Option         | Type   | Description                                                                      | Default     |
+|----------------|--------|----------------------------------------------------------------------------------|-------------|
+| `executor`     | string | Unique executor name. See the list of possible values below.                     | -           |
+| `startTime`    | string | Time offset since test start this scenario should begin execution.               | `"0s"`      |
+| `gracefulStop` | string | Time to wait for iterations to finish executing before stopping them forcefully. See the [gracefulStop](#graceful-stop-and-ramp-down) section. | `"30s"`     |
+| `exec`         | string | Name of exported JS function to execute.                                         | `"default"` |
+| `env`          | object | Environment variables specific to this scenario.                                 | `{}`        |
+| `tags`         | object | [Tags](/using-k6/tags-and-groups) specific to this scenario.                     | `{}`        |
+
+Possible values for `executor` are the executor name separated by hyphens:
+- `shared-iterations`
```

Comment:
If possible, these should be links to the section anchors below

Reply:
Sure, good idea. Resolved by a57a3c1.